src/qml/filters/audio_lowpass/ui.qml

```diff
@@ -21,9 +21,10 @@ import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.12
 import Shotcut.Controls 1.0 as Shotcut
 
-Item {
+Shotcut.KeyframableFilter {
     width: 350
     height: 125
+    keyframableParameters: ['0', '1', 'wetness']
```

Comment:
Since you are setting this variable from `KeyframableFilter` you should also initialize `startValues`, `middleValues`, and `endValues`. This especially important to give convenient default values for simple keyframes.

Reply:
Also, usually when the code gets more complicated like this, and MLT property names are abstract like `0` and `1` we like to make QML properties to give them a nicer name:
```
property string cutoffProperty: '0'
property string stagesProperty: '1'
```

Reply:
Another thing I often do is since they are equivalent:
`keyframableParameters: preset.parameters`

Reply:
> Since you are setting this variable from `KeyframableFilter` you should also initialize `startValues`, `middleValues`, and `endValues`. This especially important to give convenient default values for simple keyframes. I recommend defaults for wetness to be start=0 middle=1 and end=1.

I'll add those parameters but I don't understand the "defaults" part. It seems that these values are initialized by `KeyframableFilter` anyway if simple keyframes are active: https://github.com/mltframework/shotcut/blob/493c9c0373a462c3e0f3b158876867608c3a44d1/src/qml/modules/Shotcut/Controls/KeyframableFilter.qml#L81-L88
I tried to set the values to
```
    startValues: [0, 0, 0]
    middleValues: [0, 0, 1]
    endValues: [0, 0, 1]
```
and
```
    startValues: [0, 0, 0]
    middleValues: [0, 0, 0]
    endValues: [0, 0, 0]
```
and I don't see any difference. So probably simply initializing the arrays is enough?

Reply:
No, no other change is required. I just tested with the following, and it works as I intended:
```
    startValues: [0, 0, 0]
    middleValues: [0, 0, 1]
    endValues: [0, 0, 0]
```

Reply:
Okay, it's a bit hard to understand from the code. I'll change it then!